client/src/components/shared/Cubic.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+const Cubic = ({ lineIndex, pointIndex, reducers, ...props }) => {
+  return (
+    <g className="ad-Anchor">
+      <line
+        className="ad-Anchor-line"
+        x1={props.p1x}
+        y1={props.p1y}
+        x2={props.x1}
+        y2={props.y1}
+      />
+      <line
+        className="ad-Anchor-line"
+        x1={props.p2x}
+        y1={props.p2y}
+        x2={props.x2}
+        y2={props.y2}
+      />
+      <circle
+        className="ad-Anchor-point"
+        onMouseDown={(e) =>
+          reducers.setDraggedCubic({ lineIndex, pointIndex, anchor: 0 })
```

Comment:
In the PR description you said `setDraggedCubic` reducer sets the `draggedCubic` to `1` or `2`.
Here it is hardcoded as `0` and `1`.

Reply:
Haha a classic programmer mistake on my part. When I said `1` or `2`, I meant 'first or second' which corresponds to 0 or 1 in terms of indexes.